setup.cfg

```diff
@@ -6,7 +6,7 @@ universal=1
 
 [flake8]
 max-line-length = 120
-exclude = build,examples
+exclude = build,examples,kale/utils/print.py,tests/utils/test_print.py
```

Comment:
Is it necessary to update flake8 this time, in accordance with issue #323? This update mainly involves moving namespace from T0* to T2* to avoid collision with other libraries using the same error code.
If so, all maintainers need to reinstall flake by running `pip install -e .[dev]`.

Reply:
Shall we freeze the version 3.8.4 of flake8? It was released in Oct 2020, is it too old? I did not have any setup for pre-commit but printing errors will be reported here when upgrading to Python 3.7. We need to consider the compatibility for the future.